src/pipecat/frames/frames.proto

```diff
@@ -35,10 +35,15 @@ message TranscriptionFrame {
   string timestamp = 5;
 }
 
+message JsonFrame {
```

Comment:
I'd rename this to `MessageFrame` instead.

Reply:
Done.